ivy/functional/backends/jax/extensions/creation.py

```diff
@@ -20,8 +20,22 @@ def triu_indices(
     /,
     *,
     device: jaxlib.xla_extension.Device,
-) -> Tuple[JaxArray]:
+) -> Tuple[JaxArray, ...]:
```

Comment:
I'm not sure what `...` should be indicating in the type hinting here, can you elaborate more why we should be adding `...` to return type hint?

Reply:
So Tuple[<type>] is a hint for a tuple with a single element of type <type>,
Tupe[<type>, ...] is hint for a tuple with one or multiple elements with type <type>